server/odc-service/src/main/java/com/oceanbase/odc/service/schedule/model/ScheduleTaskType.java

```diff
@@ -38,6 +42,29 @@ public enum ScheduleTaskType {
 
     LOGICAL_DATABASE_CHANGE,
 
-    LOAD_DATA
+    LOAD_DATA;
+
+    public boolean isExecuteInTaskFramework() {
```

Comment:
if we can avoid this switch case, cause when new job type added, we will always search the code, and find where the code should be add and which value should be returned. If code not added here, default value returned, and code may not provided any waring. 
Restraint code for user should be designed, whether annotation or interface, make problem found on compile or runtime period.